lesson2/homework/helloworld/src/main/java/Account.java

```diff
@@ -36,8 +36,8 @@ public void deposit_money(long deposit){
 
 class CheckingAccount extends Account {
 
-    public CheckingAccount(long new_amount) { //returns null -- why doesn't it return the toString method?
-        super(new_amount); //doesn't seem to do anything, but I get errors when I take this line aways
```

Comment:
This is the correct code! `super()` calls your parent class's constructor.  In this case, `super(new_amount)` calls the constructor of Account.  This way, only your parent Account class has to implement setting the `amount` as the `new_amount`.  So the child class, CheckingAccount, only has to call its parent constructor with its amount.